frontend/src/components/LandingPage/TeamPage/TeamPage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
Is this code different then what exists in the current version of TeamPage.js?

Reply:
yes it's different. I deleted that code because I accidentally coded that on the wrong branch!

Reply:
The code in this version of TeamPage.js is outdated, I accidentally coded on the wrong branch!